src/routes/dashboardRoute.ts

```diff
@@ -67,4 +67,14 @@ router.get('/dashboard/:dashboardId', async (req, res) => {
     }
 });
 
+router.post('/watch', async (req, res) => {
```

Comment:
What exactly does watch mean here? this gives permission to a user who opened a link right? either change the name or leave a comment about this